requirements-dev.txt

```diff
@@ -0,0 +1,15 @@
+# Pre-commit framework
+pre-commit>=3.5.0
+
+# Testing
+pytest>=7.4.0
+
+# Python development tools
+pylint>=3.0.0
+
+# Optional: For IDE integration and manual use (pre-commit handles these automatically)
```

Comment:
Are we going to use this? they are commented out

Reply:
Ah I suppose they should be uncommented for folks who will use manually